### DISCUSSION TITLE
Just wanted to say....

### DISCUSSION THREAD

**Discussion by nevotheless** [codemagic-ci-cd/codemagic-docs#2189] [Ideas]
### Use case

```Text
I miss the slack chat :(
```

### Idea

```Text
Slack doesn't provide good tools for customer support, but there're alternatives like discord which has a richer "forum" feature which you can customize to easily create some kind of customer support area with the benefit of having the direct contact to the customers which I personally liked very much and miss it everytime i have a question.

Barrier to ask for help has risen alot with the github discussions. 

I just miss it. Much love and keep it up team.
```

**Comment by mikhail-tokarev** [comment]
Hi Tim! Thank you for the kind words! we also miss chat with our users 😊 and just recently discussed this topic internally again and probably we should create Discord community indeed. So messages like this are important and appreciated!

**Comment by kevin-suhajda** [comment]
Hey @nevotheless!!!

We decided to set up a Codemagic Community Discord server! 

There’s no message limit or chat history that disappears after 90 days, so we think it will make a great place to hang out and connect with the Codemagic Team and other Codemagic users all over the world. In addition to general chat and Community Support, you’ll also be able to see what events we’ll be attending and sponsoring in the coming months. 

## What is Discord?
[Discord](https://discord.com/) is a popular community platform where members can interact and engage with one another via text, video calls, and voice chats.

## How do I access Discord?
Joining Discord is simple. Here’s the process in five steps:

[Download the application](https://support.discord.com/hc/en-us/articles/360033931551-Getting-Started) or use a [web browser](https://discord.com/login). 
Note for Mac users, Discord asks for permission to receive keystrokes — which improves accessibility by enabling systemwide push-to-talk (PTT) to work. If you disable this permission, Discord will still function just fine, you’ll just need to open the app for your PTT hotkey to function.
[Create an account](https://discord.com/register) If you have a personal account, you can create another profile, or professional account, and switch between the two as you’d like.
Check your [privacy settings](https://discord.com/safety/360043857751-four-steps-to-a-super-safe-account).
Configure your [notifications](https://support.discord.com/hc/en-us/articles/215253258-Notifications-Settings-101).
Join the [Codemagic Discord Server](https://discord.gg/KPta9HrX).

Having trouble installing the application or creating an account? Visit the [Discord Help Center](https://support.discord.com/hc/en-us).

We look forward to seeing you on Discord soon!
